DTS-STN/canadian-dental-care-plan
# AB#2997 Intake - Applicant Information Screen 4.0

### Description
- add Screen 4.0 on Intake (Applicant Information)
- add mock handler for marital status lookup
- add locales
- add Luhn's Algorithm to validate SIN
- seed faker db with marital statuses
- add method in lookup service to get all marital statuses

### Related Azure Boards Work Items
[AB#2997](https://dev.azure.com/DTS-STN/1fc40a8f-28cf-47bc-b6e4-1c234bd06177/_workitems/edit/2997), [AB#2929](https://dev.azure.com/DTS-STN/1fc40a8f-28cf-47bc-b6e4-1c234bd06177/_workitems/edit/2929)

### Screenshots (if applicable)
![image](https://github.com/DTS-STN/canadian-dental-care-plan/assets/48450599/59e697ef-80b0-4840-92aa-f7c8fe70fdb5)

![image](https://github.com/DTS-STN/canadian-dental-care-plan/assets/48450599/9cabb183-a589-41c6-8261-592c8cc65dbb)

![image](https://github.com/DTS-STN/canadian-dental-care-plan/assets/48450599/1841ec89-a08d-4fbb-9294-7ae3d87eb3fe)



### Checklist


### Test Instructions
- go to `/intake`
- for now go to `intake/${id}/applicant-information`
- fill out a valid SIN (046454286)
- fill out last name
- check marital status
- -submit

if (married or common-law is selected, user is navigated to `intake/${id}/spouse-information` otherwise `intake/${id}/contact-information`

### Additional Notes
- I will be adding in Screen 4.b (Spouse Information) in a subsequent PR
